rsc/src/main/scala/rsc/scalasig/Writer.scala

```diff
@@ -28,32 +34,53 @@ final class Writer private (settings: Settings, reporter: Reporter, infos: Infos
     }
     val moduleSym = if (sym.desc.isTerm) sym else companionSym
 
-    if (done(sym)) return
-    val pickle = Pickle(settings, mtab, sym, companionSym)
-    pickle.emitEmbeddedSym(sym, ToplevelMode)
-    done += sym
-    if (mtab.contains(companionSym)) {
-      pickle.emitEmbeddedSym(companionSym, ToplevelMode)
-      done += companionSym
+    if (done(sym)) return timings
```

Comment:
I'm not a fan of using return in Scala but in this case it looks like you are matching the existing style.